experimental/cpu_ukernel/system_plugin/builtin_ukernel_system_plugin.c

```diff
@@ -0,0 +1,140 @@
+// Copyright 2023 The IREE Authors
```

Comment:
we should be able to use the same plugin.c for both - the only difference being the assert printf could be handled with an #ifdef on IREE_UKERNEL_STANDALONE or some other define set in the build files